app/controllers/ConfigController.scala

```diff
@@ -56,34 +57,34 @@ class ConfigController @Inject() (implicit val env: Environment[User, SessionAut
   def getCityAPIDemoParams() = Action.async { implicit request =>
     val mapboxApiKey: String = Play.configuration.getString("mapbox-api-key").get
     val cityStr: String = Play.configuration.getString("city-id").get
-    val southwestLat: Double = Play.configuration.getDouble("city-params.southwest-boundary-lat." + cityStr).get
-    val southwestLng: Double = Play.configuration.getDouble("city-params.southwest-boundary-lng." + cityStr).get
-    val northeastLat: Double = Play.configuration.getDouble("city-params.northeast-boundary-lat." + cityStr).get
-    val northeastLng: Double = Play.configuration.getDouble("city-params.northeast-boundary-lng." + cityStr).get
+    val southwestLat: Double = ConfigTable.getSouthwestLat
+    val southwestLng: Double = ConfigTable.getSouthwestLng
+    val northeastLat: Double = ConfigTable.getNortheastLat
+    val northeastLng: Double = ConfigTable.getNortheastLng
 
-    val attributeCenterLat: Double = Play.configuration.getDouble("city-params.api-demos.attribute-center-lat." + cityStr).get
-    val attributeCenterLng: Double = Play.configuration.getDouble("city-params.api-demos.attribute-center-lng." + cityStr).get
-    val attributeZoom: Double = Play.configuration.getDouble("city-params.api-demos.attribute-zoom." + cityStr).get
-    val attributeLat1: Double = Play.configuration.getDouble("city-params.api-demos.attribute-lat1." + cityStr).get
-    val attributeLng1: Double = Play.configuration.getDouble("city-params.api-demos.attribute-lng1." + cityStr).get
-    val attributeLat2: Double = Play.configuration.getDouble("city-params.api-demos.attribute-lat2." + cityStr).get
-    val attributeLng2: Double = Play.configuration.getDouble("city-params.api-demos.attribute-lng2." + cityStr).get
+    val attributeCenterLat: Double = ConfigTable.getApiAttributeCenterLat
```

Comment:
Let's make this look a bit cleaner by adding another case class, call it something like
```
case class MapParams(centerLat: Float, centerLng: Float, zoom: Float, ...)
```

Then you can just have a `getAPIAttributeMapParams` function instead of the 6 separate getters.